### DISCUSSION TITLE
[PSD6] Exercise: T11 - Consensus

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#142] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210015355-a7bc1088-081b-402d-9d18-5253aa631daa.png)

1. Prove T11 using perfect induction

2. Prove T11 using other theorems and axioms

**Comment by Elia-Belli** [comment]
- B C D X Y
0 0 0 0 0
0 0 1 1 1
0 1 0 0 0
0 1 1 1 1
1 0 0 0 0
1 0 1 0 0
1 1 0 1 1
1 1 1 1 1

- $BC + \bar B D + CD$
$BC + \bar B D + (B + \bar B) CD$
$BC + \bar B D + BCD + \bar B C D$
$\bar B (D+CD) + BC (1+D)$
$\bar B (D (1+C)) + BC$
$\bar B D + BC$

**Comment by FeddyLix17** [comment]
corretto
